src/Components/PrivateRoute/PrivateRoute.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { navigate, useNavigate, Route, Navigate } from "react-router-dom";
+
+
+function PrivateRoute({ children }) {
+  const navigate = useNavigate();
```

Comment:
I don't think so you are using navigate here, so you can remove it.

Reply:
I will use it in next pr for auth